src/test/features/dashboard/helpers/claimStatusFlow.ts

```diff
@@ -1,135 +1,250 @@
+          ...baseFullAdmissionData
+        }
+      }
+
+      context('when a claim is in pay immediately', () => {
+        const payImmediatelyClaim = {
+          ...fullAdmissionClaim,
+          response: { ...fullAdmissionClaim.response, ...basePayImmediatelyData }
+        }
+
+        it('should extract the correct dashboard if the payment is before deadline', () => {
+          expect(ClaimStatusFlow.dashboardFor(new Claim().deserialize(payImmediatelyClaim))).to.eql('pay_immediately')
+        })
+
+        it('should extract the correct dashboard if the payment is after deadline', () => {
+          const afterDeadlineClaim = {
+            ...payImmediatelyClaim,
+            responseDeadline: MomentFactory.currentDate().subtract(1, 'days')
+          }
+          expect(ClaimStatusFlow.dashboardFor(new Claim().deserialize(afterDeadlineClaim))).to.eql('payment_past_deadline')
+        })
+      })
+
+      it('should extract the correct dashboard for pay by set date', () => {
+        const payBySetDateClaim = {
+          ...fullAdmissionClaim,
+          response: { ...fullAdmissionClaim.response, ...basePayBySetDateData }
+        }
+        expect(ClaimStatusFlow.dashboardFor(new Claim().deserialize(payBySetDateClaim))).to.eql('pay_by_set_date')
+      })
+
+      it('should extract the correct dashboard for pay by instalments', () => {
+        const payByInstalmentsDateClaim = {
+          ...fullAdmissionClaim,
+          response: { ...fullAdmissionClaim.response, ...basePayByInstalmentsData }
+        }
+        expect(ClaimStatusFlow.dashboardFor(new Claim().deserialize(payByInstalmentsDateClaim))).to.eql('pay_by_instalments')
+      })
+    })
+
+    context('when a claim in partial admission', () => {
+      const partAdmissionClaim = {
+        ...sampleClaimObj,
+        responseDeadline: MomentFactory.currentDate().add(1, 'days'),
+        response: {
+          ...baseResponseData,
+          ...basePartialAdmissionData
+        }
+      }
+
+      context('when a claim is in pay immediately', () => {
+        const payImmediatelyClaim = {
+          ...partAdmissionClaim,
+          response: { ...partAdmissionClaim.response, ...basePayImmediatelyData }
+        }
+
+        it('should extract the correct dashboard', () => {
+          expect(ClaimStatusFlow.dashboardFor(new Claim().deserialize(payImmediatelyClaim))).to.eql('admitted_part_claim')
+        })
+
+        context('when claimant accepts partial amount', () => {
+          const acceptedClaim = {
+            ...payImmediatelyClaim,
+            claimantResponse: baseAcceptationClaimantResponseData
+          }
+
+          it('should extract the correct dashboard if the payment is before deadline', () => {
+            expect(ClaimStatusFlow.dashboardFor(new Claim().deserialize(acceptedClaim))).to.eql('accepted_partial_amount')
+          })
+
+          it('should extract the correct dashboard if the payment is after deadline', () => {
```

Comment:
can be possible to change generic statement i.e. correct dashboard to specific what we are expecting please

Reply:
👍

Reply:
I have changed the unit tests as requested. I won't change the route tests because there's another PR open to refactor those, and the test name will be gone so it would be pointless to change those :)